Source/Plugins/SyntheticBuilding/Gui/Widgets/StatsGenAxisODFWidget.cpp

```diff
@@ -619,29 +613,28 @@ void StatsGenAxisODFWidget::on_loadODFTextureBtn_clicked()
     return;
   }
 
-    size_t numOrients = 0;
-    QString filename = file;
-    std::ifstream inFile;
-    inFile.open(filename.toLatin1().data());
+  size_t numOrients = 0;
+  std::ifstream inFile;
+  inFile.open(file.toLatin1().data());
```

Comment:
You might want to look into the qPrintable for const char* conversion.  It does operate slightly differently as it doesn't force Latin1 characters, but if that is acceptable, I find it a lot more readable and easier to remember than whether I'm using toLatin1().data() or toLocal8Bit().data().  I wouldn't consider it important for right now, though.